README.md

```diff
@@ -1,44 +1,99 @@
-![MARS-curiosity logo](https://www.andreamagni.eu/images/MARS-Curiosity-d.png)
+![MARS-curiosity logo](media/logo-small-MARS.png)
 
-_\* The Delphi stylized helmet icon is trademark of Embarcadero Technologies._
```

Comment:
As you still have that image in the logo I think the comment should stay as to not infringe the copyright.